portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Opens a modal with the given image and caption.
+ */
+function openSimpleImageModal(modalId, modalImgId, captionId, closeId, imgSrc, captionText) {  
+  var modal = document.getElementById(modalId);
```

Comment:
let modal = ...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations

Reply:
Ah yes will change this, is it okay to use const given it is part of ES6?

Reply:
sure

Reply:
Done